src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClientExtensions.cs

```diff
@@ -27,8 +27,9 @@ internal static class AuthorizationClientExtensions
     {
         public const string CustomRole = "CustomRole";
         public const string AuthorizationDeniedException = "Authorization_RequestDenied";
+		public const string DeletedObject = "Object Deleted";
```

Comment:
@darshanhs90 same comment about the indentation being a little off in the added code